### PR TITLE
Add suppressed G51 layout.

### PR DESCRIPTION
**Pull request by tijon1**
check discord for sound

dedicated/listen server has been tested for netcode, all four gamestyles are working correctly as intended.

### CODE REVIEW

File: BWBP_SKC_Pro/Classes/G51WeaponParamsTactical.uc [hunk 1]
```diff
@@ -321,7 +355,7 @@ defaultproperties
 		WeaponBoneScales(2)=(BoneName="HoloSightUpper",Slot=55,Scale=0f)
 		WeaponBoneScales(3)=(BoneName="HoloSightLower",Slot=56,Scale=0f)
 		SightOffset=(X=3.000000,Y=-0.01000,Z=0.500000)
-		GunAugments(0)=(GunAugmentClass=class'BallisticProV55.Augment_IRScope',BoneName="tip",Scale=0.05,AugmentOffset=(x=-27,y=-2.6,z=0),AugmentRot=(Pitch=0,Yaw=0,Roll=-16384))
+		GunAugments(0)=(GunAugmentClass=class'BallisticProV55.Augment_IRScope',BoneName="tip",Scale=56666666666,AugmentOffset=(x=-27,y=-2.6,z=0),AugmentRot=(Pitch=0,Yaw=32768,Roll=-16384))
```
Comment (tijon1) [inline BWBP_SKC_Pro/Classes/G51WeaponParamsTactical.uc]: needs to be fixed

File: BWBP_SKC_Pro/Classes/G51WeaponParamsRealistic.uc [hunk 2]
```diff
@@ -330,12 +367,15 @@ defaultproperties
...
+		//Attachments
+			
 		//Attachments
 		WeaponBoneScales(0)=(BoneName="IronsLower",Slot=53,Scale=0f)
-		WeaponBoneScales(1)=(BoneName="CarryHandle",Slot=54,Scale=-1)
+		WeaponBoneScales(1)=(BoneName="CarryHandle",Slot=54,Scale=1f)
 		WeaponBoneScales(2)=(BoneName="HoloSightUpper",Slot=55,Scale=0f)
 		WeaponBoneScales(3)=(BoneName="HoloSightLower",Slot=56,Scale=0f)
-		SightOffset=(X=3.000000,Y=-0.01000,Z=0.500000)
+		SightOffset=(X=-0.500000,Y=0.00000,Z=0.90)
```
Comment (tijon1) [inline BWBP_SKC_Pro/Classes/G51WeaponParamsRealistic.uc]: needs fix

File: BWBP_SKC_Pro/Classes/G51WeaponParamsRealistic.uc [hunk 3]
```diff
@@ -330,12 +367,15 @@ defaultproperties
 		Weight=5
 		LayoutName="IR Scope"
 		LayoutTags="IR"
+		//Attachments
+			
 		//Attachments
 		WeaponBoneScales(0)=(BoneName="IronsLower",Slot=53,Scale=0f)
-		WeaponBoneScales(1)=(BoneName="CarryHandle",Slot=54,Scale=-1)
+		WeaponBoneScales(1)=(BoneName="CarryHandle",Slot=54,Scale=1f)
```
Comment (tijon1) [inline BWBP_SKC_Pro/Classes/G51WeaponParamsRealistic.uc]: needs fix

**Review by tijon1 — COMMENTED** [review]
fixed slipups

**Review by MarcMoylan — APPROVED** [review]
be sure to check the ADS times and other functions that are inherited when you copy and paste subobjects

**Pull request merged**